polaris-for-vscode/tsconfig.json

```diff
@@ -1,18 +1,8 @@
 {
-  "compilerOptions": {
-    "module": "commonjs",
-    "target": "es2020",
-    "lib": ["es2020"],
-    "outDir": "dist",
-    "rootDir": "src",
-    "sourceMap": true,
-    "skipLibCheck": true,
-    "composite": true,
-    "incremental": true,
-    "declaration": true,
-    "declarationMap": true
-  },
-  "include": ["src"],
-  "exclude": ["node_modules", "dist"],
-  "references": [{"path": "./client"}, {"path": "./server"}]
+  "files": [],
+  "references": [
+    {"path": "./client"},
+    {"path": "./server"},
+    {"path": "../polaris-tokens"}
```

Comment:
**question:** Can you help me understand why we include the `../polaris-tokens` reference?

Reply:
Since `@shopify/polaris-tokens` is a dependency of `polaris-for-vscode`, we should let TypeScript know that it should build the types for tokens as well when running `tsc -b`. That's what I'm gathering based on my understanding of the [project references docs](https://www.typescriptlang.org/docs/handbook/project-references.html#what-is-a-project-reference), but I could be interpreting it incorrectly 😓

Reply:
Removed the reference entirely. Seems we could only need it if we wanted to build (as you mentioned) directly using `tsc -p polaris-for-vscode/tsconfig.json` but I think we should keep it simple to using turbo for building from the root (ex: `yarn turbo run build --filter=polaris-for-vscode`) which will build with the proper dependency order (including types) 🏗️ 👍